packages/binaryen/binaryen.0.14.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+synopsis: "OCaml bindings for Binaryen"
+maintainer: "oscar@grain-lang.org"
+authors: "Oscar Spencer"
+license: " Apache-2.0"
+homepage: "https://github.com/grain-lang/binaryen.ml"
+bug-reports: "https://github.com/grain-lang/binaryen.ml/issues"
+depends: [
+  "ocaml" {>= "4.12"}
+  "dune" {>= "2.9.1" & < "3.0.0"}
```

Comment:
Is this a conservative bound or the package is actually incompatible with dune 3?

Reply:
We're being conservative because I haven't put this though the paces. I plan to do that testing and expand the range together with jsoo 4 and the binaryen 106 release. How does that sound?